### PR TITLE
Update themes.yml

### PR DESCRIPTION
**Pull request by maletor**

### CODE REVIEW

File: data/themes.yml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 - { colors: '#4D394B,#3E313C,#4C9689,#FFFFFF,#3E313C,#FFFFFF,#38978D,#EB4D5C', name: 'Aubergine' }
 - { colors: '#1d1d1d,#000000,#0ba8ca,#FFFFFF,#075566,#FFFFFF,#0ba8ca,#EB4D5C', name: 'B-MIND' }
+- { colors: '#181818,#585858,#7cafc2,#f8f8f8,#b8b8b8,#f8f8f8,#f7ca88,#ab4642', name: "Base16 Default Dark' }
```
Comment (paracycle) [inline data/themes.yml]: There is a quote style mismatch on this line that is breaking the build: `"Base16 Default Dark'`.

Comment (paracycle) [inline data/themes.yml]: Please use single quotes everywhere.

**Comment by maletor** [comment]
There ya go.

**Comment by paracycle** [comment]
Thanks.

**Pull request merged**